### PR TITLE
Changed "welcome page" template and CSS so it fits on HD monitors in fullscreen without scrollbar.

### PR DESCRIPTION
**Pull request by mateuszbieniek**
This PR changes a little "welcome page" template, so it fits without scrolling on FullHD monitors (in a maximized window, tested on Firefox & Chrome). 

Also, it makes the footer stick to the bottom on bigger screens.

### CODE REVIEW

File: assets/scss/welcome-page.scss [hunk 1]
```diff
@@ -30,8 +32,9 @@ a {
     }
 }
 .footer {
-    background: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSI1MTQuNTg5IiBoZWlnaHQ9IjUxNC4yNTQiIHZpZXdCb3g9IjAgMCA1MTQuNTg5IDUxNC4yNTQiPjxkZWZzPjxzdHlsZT4uYXtmaWxsOiNkN2Q4ZTI7b3BhY2l0eTowLjI7fTwvc3R5bGU+PC9kZWZzPjxwYXRoIGNsYXNzPSJhIiBkPSJNNTE2LjI1OCwzNDUuN1YyMTQuNzE4aC0zMTguN1Y1MzMuNDA2SDMyOS40Njl2NTcuNDM4SDU3NC42VjM0NS43Wk0zMjkuNDY5LDUwMi41MTdIMjI4LjQ0OVYyNDUuNTg2SDQ4NS4zNjJWMzQ1LjdIMzI5LjQ2OVptMjE1Ljc4NSw1OC45NjhIMzU4LjhWMzc1LjA2MUg1NDUuMjU1WiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMTI0LjMyNyAtMjg0LjczMykgcm90YXRlKDMwKSIvPjwvc3ZnPg==") no-repeat 107% 0;
-    height: 370px;
+    position: absolute;
```
Comment (ciastektk) [inline assets/scss/welcome-page.scss:35]: You shouldn't add position absolute here because on smaller screen footer will overlap section with links.

Comment (mateuszbieniek) [reply assets/scss/welcome-page.scss:35]: No, it won't :) :

https://github.com/ezsystems/ezplatform/pull/529/files#diff-93e65dbd1c7c64d40913e27b6005de91R8

Comment (ciastektk) [reply assets/scss/welcome-page.scss:35]: Screen Shot 2020-04-07 at 11 58 40

Comment (mateuszbieniek) [reply assets/scss/welcome-page.scss:39]: OK, good catch! Fixed.

**Comment by lserwatka** [comment]
@alongosz could you merge it to ezplatform-ee ?

**Comment by alongosz** [comment]
Merged horizontally as ezsystems/ezplatform-ee@c0e02af79.

**Pull request merged**